hummingbot/strategy/pure_market_making/constant_size_sizing_delegate.pyx

```diff
@@ -49,13 +59,19 @@ cdef class ConstantSizeSizingDelegate(OrderSizingDelegate):
             bid_order_size = market.c_quantize_order_amount(market_info.symbol, self.order_size)
             ask_order_size = market.c_quantize_order_amount(market_info.symbol, self.order_size)
 
-        if (bid_order_size ==0):
-            self.logger().network(f"Buy(bid) order size is less than minimum order size. Buy order will not be placed",
-                                  f"The order size is too small for the market for buy order. Check order size in configuration.")
+        if self.log_warning_order_size:
```

Comment:
`cdef` functions are meant to optimize for performance. Python function calls and properties are ~1000x slower than C function calls. So you should use the fast `_` property to access the flag here.